modules/oeFax/contact.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018-2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace Modules\oeFax\Controller;
```

Comment:
Why is there a namespace declared here without a class in the script?

Reply:
Because I use to have other stuff going on in this script and forgot.